### ISSUE TITLE
Allow scale to be changed on a per-chart basis

### ISSUE THREAD

**Issue by Castavo** [aimhubio/aim#2358]
## 🚀 Feature

It would be very neat if one could change the axis scales (linear or log) for only one chart at a time.

### Motivation

That would prove especially useful when one of the metrics used happen to be negative. I have an exponential metric (which would be neat to see in y log-scale) and a negative metric, which I do not want to apply log-scale to.

### Pitch

Ideally it would be lovely to have a toggle near the axis of a chart, to switch it to log-scale. 

### Alternatives

* It could also be handy to have a burger menu per chart, with one of the option being to change a scale.
* When non-applicable, the log-scale could simply be ignored instead of replacing the graph by an error message.
* An option in the "Axes scale" popup could have a text input which would allow for filtering among the charts by metric for example.

**Comment by gorarakelyan** [comment]
@Castavo makes a lot of sense. Are there any other chart controls, besides log scale switcher, that you would like to change per chart basis?

**Comment by Castavo** [comment]
Yes ! I was also thinking about a per-chart smoothing policy, my training metric is very noisy and needs the smoothing, while the validation metrics suffer from it, so I have to alternate between smoothing on and off to see the curves clearly...

On another, it could be nice also to apply logs or smoothing depending on a field, like `metric.name` or `metric.context.subset`.
